bundler/lib/bundler/cli.rb

```diff
@@ -316,28 +316,11 @@ def update(*gems)
       D
       method_option "paths", :type => :boolean,
                              :banner => "List the paths of all gems that are required by your Gemfile."
+      method_option "verbose", :type => :boolean,
+                               :banner => "Show verbose output."
       method_option "outdated", :type => :boolean,
                                 :banner => "Show verbose output including whether gems are outdated."
       def show(gem_name = nil)
-        if ARGV[0] == "show"
-          rest = ARGV[1..-1]
-
-          if flag = rest.find{|arg| ["--verbose", "--outdated"].include?(arg) }
-            Bundler::SharedHelpers.major_deprecation(2, "the `#{flag}` flag to `bundle show` was undocumented and will be removed without replacement")
-          else
-            new_command = rest.find {|arg| !arg.start_with?("--") } ? "info" : "list"
-
-            new_arguments = rest.map do |arg|
-              next arg if arg != "--paths"
-              next "--path" if new_command == "info"
-            end
-
-            old_argv = ARGV.join(" ")
-            new_argv = [new_command, *new_arguments.compact].join(" ")
-
-            Bundler::SharedHelpers.major_deprecation(2, "use `bundle #{new_argv}` instead of `bundle #{old_argv}`")
-          end
-        end
```

Comment:
This whole block of code is wrapped under a `unless Bundler.feature_flag.bundler_3_mode?` condition. We need to remove that condition so that the command doesn't get removed when we release bundler 3.

Reply:
ok, will do that, thanks for the heads up ;)

Reply:
Thanks!